content/blog/zenbleed-patch-call-for-testing.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Testers needed: Zenbleed patch for AlmaLinux 8 and 9"
+type: blog
+author: 
+ name: "Andrew Lukoshko"
+ bio: "Release Engineering Lead"
+ image: /users/alukoshko.jpg
+date: '2023-07-24'
+post:
+    title: "Testers needed: Zenbleed patch for AlmaLinux 8 and 9"
+    image: /blog-images/23.07.24.zenbleed.png
+---
+
+Earlier today our community pointed out a new, trivially exploitable flaw in [AMD systems called Zenbleed](https://lwn.net/Articles/939099/). Due to an accident on the AMD side, the patch was released ahead of responsible disclosure, and unpatched systems are at great risk. We were able to pull in the patch, get through our normal testing, and we are now ready for wider testing for both AlmaLinux 8 and 9.
+
+
+## How did AlmaLinux get the patch?
+
+The fix was released by AMD, so we were able to pull that directly in, similar to what all other distributions are currently having to do. We pulled in three patches from linux-firmware upstream:
+
+[https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/commit/amd-ucode?id=69143e8eca62a80b9791b8d358d1cc4c90e373c9](https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/commit/amd-ucode?id=69143e8eca62a80b9791b8d358d1cc4c90e373c9)
+
+[https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/commit/amd-ucode?id=b250b32ab1d044953af2dc5e790819a7703b7ee6](https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/commit/amd-ucode?id=b250b32ab1d044953af2dc5e790819a7703b7ee6)
+
+[https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/commit/amd-ucode?id=0bc3126c9cfa0b8c761483215c25382f831a7c6f](https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/commit/amd-ucode?id=0bc3126c9cfa0b8c761483215c25382f831a7c6f)
+
+You can see the diff of the changes on [git.almalinux.org](https://git.almalinux.org/rpms/linux-firmware/commit/b908df53b1c732aa18435a947f8e3c6a45ebc72c). 
+
+
+## How do I install the updated packages?
+
+Due to the risks involved in these patches, these packages are not yet in production and need testing! If you are willing to help provide us feedback, and have access to a **bare metal AMD system**, you can manually install them by pulling them from the AlmaLinux Build System.  
+
+To install the new RPM on AlmaLinux 8: 
+
+> dnf update https://build.almalinux.org/pulp/content/builds/AlmaLinux-8-x86_64-7032-br/Packages/l/linux-firmware-20230404-114.git2e92a49f.el8_8.alma.noarch.rpm
+
+For AlmaLinux 9:
+
+> dnf update https://build.almalinux.org/pulp/content/builds/AlmaLinux-9-x86_64-7033-br/Packages/l/linux-firmware-20230310-134.el9_2.alma.noarch.rpm
```

Comment:
We should be consistent for all commands / snippets. I propose we change this and the one above accordingly. 

`dnf update https://build.almalinux.org/pulp/content/builds/AlmaLinux-9-x86_64-7033-br/Packages/l/linux-firmware-20230310-134.el9_2.alma.noarch.rpm`